### DISCUSSION TITLE
需要基于xSwitch代理线上css文件，但是umi4的项目启动后，会有多个css，这种情况怎么处理？

### DISCUSSION THREAD

**Discussion by Cherry4693** [umijs/umi#11537] [Q&A]
以前是只有一个umi.css文件，可以基于文档中的进行代理
[https://umijs.org/docs/guides/debug#xswitch](url)

现在多个css该怎么代理？是否可以通过配置，本地启动时，是一个css文件？
![image](https://github.com/umijs/umi/assets/67308487/664a04c2-c4be-46ce-a442-efb5181f4f07)

**_ extraBabelPlugins: process.env.NODE_ENV === 'production' ? ['babel-plugin-dynamic-import-node'] : [],_
_配置后，只是js文件不再分包，对css文件无效。_**

**Comment by fz6m** [comment]
自己想办法配一个通用的代理方案，比如把访问某个域名的请求都代理到本地 umi 启动的端口，这样所有的资源都会代过去了。可以用 charles 、[switch omega](https://chrome.google.com/webstore/detail/proxy-switchyomega/padekgcemlokbadohgkifijomclgjgif) 、[whistle](https://github.com/avwo/whistle) 等工具做代理。

如果短期没法解决，可以升级到最新的 umi 版本（ `>= 4.0.76` ）先使用如下配置：

```ts
export default {
  ...(process.env.NODE_ENV === 'development' ? {
      routeLoader: {
        moduleType: 'cjs'
      },
      mfsu: false,
      styleLoader: {},
  } : {})
}
```

这可以让 umi 不再默认按页分包，并且 css 都内置到 js 里，这样你只需要代理 `umi.js` 就行了，但是未来随着项目增大，开发会越来越慢，不建议这么做，还是要寻找一种通用代理方法。

**Reply by Cherry4693** [reply]
谢谢
